cranelift/fuzzgen/src/lib.rs

```diff
@@ -224,14 +273,14 @@ where
         Ok(ctx.func)
     }
 
-    fn generate_func(&mut self) -> Result<Function> {
-        let func = FunctionGenerator::new(&mut self.u, &self.config).generate()?;
+    pub fn generate_func(&mut self, target_triple: Triple) -> Result<Function> {
+        let func = FunctionGenerator::new(&mut self.u, &self.config, target_triple).generate()?;
         self.run_func_passes(func)
     }
 
     /// Generate a random set of cranelift flags.
     /// Only semantics preserving flags are considered
-    fn generate_flags(&mut self) -> Result<Flags> {
+    pub fn generate_flags(&mut self, target_arch: Architecture) -> Result<Flags> {
```

Comment:
I think this doesn't need to be `pub`; it looks like it's only called from this module now.